derrida/books/templates/books/public_canvas_detail.html

```diff
@@ -362,7 +362,8 @@ <h3 class="gallery-image-view__list-title--bold">Admin Actions</h3>
 
       <div class="annotation-selections">
         {# large size image #}
-        <img class="gallery-image-view__img" src="{% url 'books:canvas-image' instance.slug canvas.short_id 'large' %}" />
+        <img class="gallery-image-view__img" src="{% url 'books:canvas-image' instance.slug canvas.short_id 'large' %}"
+          src="{% url 'books:canvas-image' instance.slug canvas.short_id 'large' '@2x' %} 2x" />
```

Comment:
ought this also be `srcset`, as in `gallery-image.html`? seems a little weird to define two `src` attributes.